pkg/controller/nginxingresscontroller/utils.go

```diff
@@ -66,8 +66,8 @@ func generatePodArgs(instance *k8sv1alpha1.NginxIngressController) []string {
 	if instance.Spec.NginxStatus != nil && instance.Spec.NginxStatus.Enable {
 		args = append(args, "-nginx-status")
 
-		if instance.Spec.NginxStatus.Port != 0 {
-			args = append(args, fmt.Sprintf("-nginx-status-port=%v", instance.Spec.NginxStatus.Port))
+		if instance.Spec.NginxStatus.Port != nil && *instance.Spec.NginxStatus.Port != 0 {
```

Comment:
do we need to check for `instance.Spec.NginxStatus.Port != 0 `? since we enforce in OpenAPI the range.

Reply:
Yeah I did this just in case, but I guess we can trust in openapi! 💪